cumulusci/robotframework/Salesforce.py

```diff
@@ -239,7 +240,8 @@ def get_record_type_id(self, obj_type, developer_name):
         soql = "SELECT Id FROM RecordType WHERE SObjectType='{}' and DeveloperName='{}'".format(
             obj_type, developer_name
         )
-        res = self.cumulusci.sf.query_all(soql)
+        with self.cumulusci._perf_wrapper():
+            res = self.cumulusci.sf.query_all(soql)
```

Comment:
I thought the goal was to have a listener that could collect performance metrics automatically without needing to remember to activate it for each call to Salesforce. Are there simple_salesforce API calls that we don't want performance metrics for?

Reply:
Well that was the goal from the point of view of the Robot programmer. It wasn't explicitly a goal for the Python programmer.

I inherited the hooking bits of the code and didn't really question it. I guess I could hook sf.session in listener.start_suite() and then unhook in close() or something like that? Knowing when to unhook is clearer now than it would have been at the start of the project when I didn't have a Listener object around listening for that event.

Reply:
Okay, fixed in the last two checkins: 3bd6399 and 
0e47c29